tools/apidiff/merger.cs

```diff
@@ -13,6 +13,14 @@ static string GetVersion (string line)
 		return line.Substring (start, end - start);
 	}
 
+	static string DestinationPath = null;
+
+	public static void Process (string platform, string path, string os, string destinationPath)
+	{
+		DestinationPath = destinationPath;
+		Process (platform, path, os);
+	}
```

Comment:
As mentioned above, this is not truly needed.

Reply:
Agreed. If you want two entry points you could just do:

```csharp
 	public static void Process (string platform, string path, string os, string? destinationPath = null)
 ```

But since you are just splatting the args to it, I'd just do:

```csharp
 	public static void Process (string platform, string path, string os, string? destinationPath)
```

and call it with:

```
 				Process (args [0], args [1], args [2], args.Length > 3 ? args [3] : null);
```

since you are going to implicitly null it out anyway

OR you could be LINQ cute with:

```
 				Process (args [0], args [1], args [2], args.ElementAtOrDefault (3));
```

which should give you null back in that case.